### ISSUE TITLE
Broken links on docs.meteor.com

### ISSUE THREAD

**Issue by glasser** [meteor/meteor#3703]
From #3643: 

> Additionally, both showdown and markdown are listed in the TOC under Packages, but the showdown listing is linked to a non-existent anchor, which scrolls one top of the page, as does the stylus listing. I also found it odd that the spiderable listing pointed to Atmosphere.

**Comment by hwillson** [comment]
Hi @glasser - I created pull request #3842 to help:

1. Showdown - I've removed the "showdown" link from the packages menu since it has been moved to the markdown package (in v0.9.4).

2. Stylus - I'm assuming MDG is no longer maintaining the Stylus package? I'm asking as I see it was removed from the packages.html file in https://github.com/meteor/meteor/commit/c06ee6d07c34359c21ee97d07959d1bc616bd870. Assuming this, I've removed it from the full API left hand nav (let me know if you instead want it preserved and I'll add it back along with the missing stylus.html description file).

3. Spiderable - I'm assuming this one should stay as is, since the code was modified to allow custom TOC links for spiderable (https://github.com/meteor/meteor/commit/104636c6a7b7b88aafc4837bdd65b4d7490ee148). 

Thanks!
